### DISCUSSION TITLE
Demo example failing. Can anyone confirm if it is ok?

### DISCUSSION THREAD

**Discussion by DanRRRR** [epochpic/epoch#484] [General]
Hello All,
Can you please run the epoch2d demo example called ionization.deck from the example_decks and confirm it works as is and by uncommenting collisional ionization? In my case it starts ok, but stops immediately. Could be the bug i introduced but problem is that all other things work ok

The only compilation options i uncommented from default were
DEFINES += $(D)PARTICLE_ID
DEFINES += $(D)PER_PARTICLE_CHARGE_MASS

**Comment by TomGoffrey** [comment]
Which version of EPOCH are you running?

**Reply by DanRRRR** [reply]
4.19 and added a fix manually discussed in #467 (which is still not finalized) thinking it was the problem

**Comment by keithbennett** [comment]
Works for me. The input deck is set to do zero time-steps by default, so needs setting "nsteps = -1"

**Reply by DanRRRR** [reply]
Keith, Are your SDF files from 0 to 50 all the same size despite of ionization?

**Reply by DanRRRR** [reply]
And how are used these parameters ?

  den_preplasma = 1.0e18
  len_scale = 5.0 * micron  ?

**Reply by keithbennett** [reply]
The input deck is only set to output grid-based quantities, so the size of these will never change. If a per-particle property is added then the file sizes increase over time.
Those two parameters are not used. Probably a leftover from a previous revision of the deck

**Reply by DanRRRR** [reply]
**Thanks** for clarification. Clearly, doctor left  few  napkins in the body after surgery :). Though look as not important these extra parameters and the setup are capable to confuse you for days as you expect everything should work and tested in demos.

**Comment by DanRRRR** [comment]
Thanks, Keith. Sorry, reading comprehension problem, misunderstood what "Maximum number of iterations" means and was confused what for it could be set  = 0?  :)))
